src/redis-benchmark.c

```diff
@@ -1854,19 +1872,19 @@ int main(int argc, char **argv) {
         }
 
         if (test_is_selected("set")) {
-            len = redisFormatCommand(&cmd,"SET key%s:__rand_int__ %s",tag,data);
+            len = redisFormatCommand(&cmd,"SET key%s:__rand_int__%s %s",tag,suffix,data);
             benchmark("SET",cmd,len);
             free(cmd);
         }
 
         if (test_is_selected("get")) {
-            len = redisFormatCommand(&cmd,"GET key%s:__rand_int__",tag);
+            len = redisFormatCommand(&cmd,"GET key%s:__rand_int__$s",tag,suffix);
             benchmark("GET",cmd,len);
             free(cmd);
         }
 
         if (test_is_selected("incr")) {
-            len = redisFormatCommand(&cmd,"INCR counter%s:__rand_int__",tag);
+            len = redisFormatCommand(&cmd,"INCR counter%s:__rand_int__%s",tag,suffix);
```

Comment:
The length of this key is not the same as the key used in GET and SET (because of the prefix "counter" here vs "key" for GET and SET tests). I think we'd need to compute the suffix length instead of just assuming it.